tests/out_invalid_network.json

```diff
@@ -1 +1 @@
-{"status":422,"message":"Validation Errors","validationErrors":[{"Key":"invalid key: Unexpected array key: intendedAudience/onlyFor","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/designKit/seo","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/designKit/ui","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/designKit/web","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/designKit/content","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/conforme/interoperabile","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/conforme/sicuro","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/conforme/privacy","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/conforme/accessibile","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/spid","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/pagopa","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/cie","Reason":""},{"Key":"invalid key: Unexpected boolean key: it/anpr","Reason":""},{"Key":"invalid key: Unexpected array key: it/ecosistemi","Reason":""}]}
+{"status":422,"message":"Validation Errors","validationErrors":[{"Key":"it/conforme/privacy: Unexpected boolean key","Reason":""},{"Key":"it/conforme/accessibile: Unexpected boolean key","Reason":""},{"Key":"it/conforme/interoperabile: Unexpected boolean key","Reason":""},{"Key":"it/conforme/sicuro: Unexpected boolean key","Reason":""},{"Key":"it/spid: Unexpected boolean key","Reason":""},{"Key":"it/pagopa: Unexpected boolean key","Reason":""},{"Key":"it/cie: Unexpected boolean key","Reason":""},{"Key":"it/anpr: Unexpected boolean key","Reason":""},{"Key":"it/ecosistemi: Unexpected array key","Reason":""},{"Key":"it/designKit/ui: Unexpected boolean key","Reason":""},{"Key":"it/designKit/web: Unexpected boolean key","Reason":""},{"Key":"it/designKit/content: Unexpected boolean key","Reason":""},{"Key":"it/designKit/seo: Unexpected boolean key","Reason":""},{"Key":"intendedAudience/onlyFor: Unexpected array key","Reason":""}]}
```

Comment:
Shouldn't this be
`{"Key":"it/conforme/privacy","Reason":"Unexpected boolean key"}`
instead of
`{"Key":"it/conforme/privacy: Unexpected boolean key","Reason":""}`
?

(for all the keys)

Reply:
good catch!!